buildkite/test-template-ci.j2

```diff
@@ -309,7 +327,15 @@ steps:
               echo "Image found"
               exit 0
             fi
-          - "docker build --file docker/Dockerfile.nightly_torch --build-arg max_jobs=16 --build-arg buildkite_commit=$BUILDKITE_COMMIT --build-arg USE_SCCACHE=1 --tag {{ docker_image_torch_nightly }} --target test --progress plain ."
+            - >
```

Comment:
NOTE: I didn't use `VLLM_USE_PRECOMPILED` here intentionally -- the gist is that I figured if we're trying to use nightly torch, we probably always want the wheels rebuilt. Happy to take input here of course.

Reply:
This is the correct assumption